src/styles/block/header.scss

```diff
@@ -0,0 +1,59 @@
+.header {
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px 0 #0000000d;
```

Comment:
The alpha value in the box-shadow should be two digits according to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules). Please update `#0000000d` to `#0000000D`.